Gruntfile.js

```diff
@@ -22,13 +22,16 @@ module.exports = function (grunt) {
                 options: {
                     reporter: "spec"
                 },
-                src: ["build/test/**/*.js"]
+                src: ["build/test/**/*Tests.js", "build/test/assert.js", "build/test/lint.js"]
             }
         },
 
         run: {
-            test: {
-                cmd: "./test/check-bin.sh"
+            testBin: {
+                exec: "./test/check-bin.sh"
```

Comment:
also this can be `cmd` instead of `exec` since we're not running multiple commands